cmd/peer-rest-client.go

```diff
@@ -591,15 +620,17 @@ func (client *peerRESTClient) BackgroundHealStatus() (madmin.BgHealState, error)
 
 // GetLocalDiskIDs - get a peer's local disks' IDs.
 func (client *peerRESTClient) GetLocalDiskIDs(ctx context.Context) (diskIDs []string) {
-	respBody, err := client.callWithContext(ctx, peerRESTMethodGetLocalDiskIDs, nil, nil, -1)
-	if err != nil {
-		return nil
+	conn := client.gridConn()
+	if conn == nil {
+		return
 	}
-	defer xhttp.DrainBody(respBody)
-	if err = gob.NewDecoder(respBody).Decode(&diskIDs); err != nil {
-		return nil
+
+	resp, err := getLocalDiskIDsHandler.Call(ctx, conn, grid.NewMSS())
+	if err != nil {
+		return
```

Comment:
missing return err ?

Reply:
Nah this function doesn't return any error @zveinn

Reply:
correct, my mistake.